students/tri_nguyen/lesson07/html_render.py

```diff
@@ -0,0 +1,159 @@
+# ------------------------------------------- #
+# Student:   Tri Nguyen
+# Instructor: Natasha Aleksandrova
+# Date:  23-Mar-2018
+# ------------------------------------------- #
+
+# Create an Element class
+
+
+class Element():
+    ''' Element class for rendering an html element '''
+
+    # Class attributes
+    tag_name = ''
+    indent = '    '  # 4 spaces
+
+    def __init__(self, content=None, **kwargs):
+        ''' initialize the Element object/instance '''
+        if content is None:
+            self.content = []
+        else:
+            self.content = [content]
+        self.kwargs = kwargs
+
+    def append(self, content):
+        ''' store content to self.content list '''
+        self.content.append(content)
+
+    def render(self, file_out, cur_ind=''):
+        ''' render the tag and the strings in the content '''
+        file_out.write('{}<{}{}>\n'.format(cur_ind, self.tag_name, self.set_attr()))
+
+        '''for attr in self.kwargs:
+            file_out.write(' {}="{}"'.format(attr, self.kwargs[attr]))
+        file_out.write('>\n')'''
```

Comment:
The convention in Python for multi-line block comments is to use the `#` character before each line and use triple double quoted strings for function/method docstrings. It's also a good idea to delete commented out code before committing, any old work will be stored in your version control history so you can always go back to it if you want